Delete/hashed private service

garcia-alejandro opened the pull request:
# Removed Hashed Private Service from the Hashed Portal

garcia-alejandro committed:
Remove unused @smontero/hashed-private-client-api dependency

garcia-alejandro committed:
Refactor downloadFile parsing logic

Simplify the downloadFile function in DetailsNFTPage by implementing a structured approach to extract metadata from the JSON-parsed file, enhancing code readability and maintainability. The update replaces ambiguous conditional checks with precise destructuring from the parsed object, ensuring more robust determination of the file type. This change is part of continuous refactoring efforts to streamline data handling within the marketplace's NFT details page.

garcia-alejandro committed:
Disable HashedPrivateApi integration across the platform

The HashedPrivateApi service has been deactivated and its related operations are now commented out throughout the application. This includes removal of login/logout functionality and associated API calls like data sharing and retrieval. Additionally, hard-coded fallbacks were added for certain methods to ensure that the application can handle the absence of the API gracefully without breaking the UI. The service file for HashedPrivateApi has also been deleted, signaling a complete move away from the dependency on this private server. Care should be taken to handle any orphaned references or dependent functionalities that might be affected by this change.

jmgayosso commented on src/pages/marketplace/NFTs/CreateNFTPage.vue:
```diff
@@ -75,7 +75,7 @@ export default {
     },
     async uploadToPrivateService ({ attributes, addressToShare }) {
```
I think we should remove all methods related with private api

garcia-alejandro replied:
Sure! I will remove it from the components

garcia-alejandro committed:
Removed deprecated private API integration

The codebase has been cleaned of references and functionalities related to a deprecated private API. This includes removal of environment variables, workflow configurations, and code segments that facilitated interactions with the private service. The removal eliminates potential security vulnerabilities, reduces complexity, and improves code maintainability by no longer supporting obsolete features. Operations relying on private data sharing have been streamlined to use current mechanisms, enhancing performance and stability.

garcia-alejandro committed:
Remove deprecated HashedPrivateApi integration

garcia-alejandro committed:
Update yarn.lock file

garcia-alejandro committed:
🔧 fix(preview.js): update Polkadot API endpoint URL to 'wss://c2.hashed.network/' for correct connection to the Polkadot network

jmgayosso reviewed (approved):
LGTM
